tools/simulator/frameworks/runtime-src/proj.ios_mac/mac/SimulatorApp.mm

```diff
@@ -205,10 +205,29 @@ - (void) parseCocosProjectConfig:(ProjectConfig*)config
     
     // set project directory as search root path
     FileUtils::getInstance()->setDefaultResourceRootPath(tmpConfig.getProjectDir());
-    
+    string solutionDir = tmpConfig.getProjectDir();
```

Comment:
It should be put before the previous line and setDefaultResourceRootPath with solutionDir